engine/src/main/java/org/terasology/engine/core/GameThread.java

```diff
@@ -23,6 +27,21 @@
     private static volatile Thread gameThread;
     private static BlockingDeque<Runnable> pendingRunnables = Queues.newLinkedBlockingDeque();
 
+    private static final Scheduler MAIN;
+
+    static {
+        MAIN = Schedulers.from(new Executor() {
+            @Override
+            public void execute(@NotNull Runnable runnable) {
```

Comment:
This is probably generated. We haven't agreed on any annotations to use, thus I think `@NotNull` should be removed.